middleman-core/lib/middleman-core/templates/shared/gitignore

```diff
@@ -16,3 +16,6 @@
 
 # Ignore .DS_store file
 .DS_Store
+
+# Ignore gems installed in local directory
+vendor/bundle/
```

Comment:
How about `/vendor/bundle/`? Vendor is a pretty common folder name and could be down in assets or node_modules paths.